apps/src/miniApps/neighborhood/NeighborhoodVisualization.tsx

```diff
@@ -1,61 +1,59 @@
+import Slider from '@code-dot-org/component-library/slider';
 import React from 'react';
 
 import MazeVisualization from '@cdo/apps/maze/Visualization';
 
+import NeighborhoodSpeedTracker from './NeighborhoodSpeedTracker';
+
 import moduleStyles from './neighborhood.module.scss';
 
 interface NeighborhoodVisualizationProps {
   className?: string;
-  isDarkMode: boolean;
+  isDarkMode?: boolean;
   useProtectedDiv?: boolean;
 }
-const ICON_PATH = '/blockly/media/turtle/';
 
 const NeighborhoodVisualization: React.FunctionComponent<
   NeighborhoodVisualizationProps
 > = ({className, isDarkMode, useProtectedDiv = true}) => {
-  const fullIconPath = isDarkMode
-    ? ICON_PATH + 'icons_white.png'
-    : ICON_PATH + 'icons.png';
+  const [sliderValue, setSliderValue] = React.useState(50);
```

Comment:
I think there's an edge case when a user is on a Neighborhood level, sets the speed to a different value (say `100`), goes to a non-Neighborhood level, then returns to a Neighborhood level. The slider is now displayed at `50`, but the tracker is still set to `100`.

Reply:
good catch!

Reply:
Updated, also added translations for the button labels